Package.swift

```diff
@@ -171,12 +179,21 @@ var package = Package(
       ]
     ),
 
+    .target(
+      name: "LoggingDependencies",
```

Comment:
Moving some core dependencies, like `Logger`, higher up so they can be used outside the `PointFree` module.